src/public/src/scripts/react/components/administrator/showCompanies.tsx

```diff
@@ -2,10 +2,8 @@ import * as React from "react";
 import {Link} from "react-router";
 import Navbar from "../../navbar/navbarSuperAdmin";
 import store from "../../../stores/companyStore";
-import {ICompany} from "../../../actions/companyActionCreator";
-
-import CompanyStore from "../../../stores/companyStore";
-// I import SessionStore from "../../../stores/sessionStore";
+import companyActionCreator,
+    {ICompany} from "../../../actions/companyActionCreator";
 
 interface IShowCompaniesState {
     companies : Array<ICompany>;
```

Comment:
Scrivo la nota qui ma mi riferisco a riga 16 e 24: `state` è derivato da React.Component? `_onChange` è necessario fare il bind con il `this`?